includes/Core/Util/BC_Functions.php

```diff
@@ -124,4 +124,76 @@ protected static function load_script_textdomain() {
 		return false;
 	}
 
+	/**
+	 * Basic implementation of the wp_sanitize_script_attributes function introduced in the WordPress version 5.7.0.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $attributes Key-value pairs representing `<script>` tag attributes.
+	 * @return string String made of sanitized `<script>` tag attributes.
+	 */
+	private static function wp_sanitize_script_attributes( $attributes ) {
```

Comment:
Let's make this `protected` like the others since `private` doesn't make it any more protected so it might set false expectations.
```suggestion
	protected static function wp_sanitize_script_attributes( $attributes ) {
```